### PR TITLE
cuOpt-Resources Content Update

### PR DESCRIPTION
**Pull request by SchultzC**
An update to cuOpt-Resources contents.  Move deployment scripts to trial zone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,58 +11,21 @@ Use of NVIDIA cuOpt is subject to the [End User License Agreement](https://docs.
 
 ## [NVIDIA cuOpt](https://developer.nvidia.com/cuopt-logistics-optimization)
 
-cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python and REST based microservice APIs
+cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python SDK and service APIs
```
Comment (ramakrishnap-nv) [inline README.md:14]: Should we remove sdk part ?

Comment (ramakrishnap-nv) [inline README.md:14]: If we remove SDK related things, should we also remove notebooks on them ?

Comment (SchultzC) [reply README.md:14]: As per the current plan, there will be a path to the SDK.  I think we should leave it.

Comment (SchultzC) [reply README.md:14]: I think we will keep a path to the SDK so it will be nice to keep the notebooks too.

**Review by rgsl888prabhu — APPROVED** [review]

**Pull request merged**